shared/cluster.go

```diff
@@ -409,3 +409,43 @@ func WriteDataPod(namespace string) (string, error) {
 
 	return RunCommandHost(cmd)
 }
+
+// Get service name. Used to work with stop/start k3s/rke2 services
+func getServiceName(product string, serviceType string) string {
```

Comment:
no need for this func!!

this u could simplify a lot to this: using * 
```
		cmd = fmt.Sprintf("timeout 2m sudo systemctl start %s*", product)

```

Reply:
need this method. tried without it and get this error message: 

```
EXECUTE: timeout 2m sudo systemctl start rke2* on 18.225.55.124 
WARN[2023-11-16T05:14:47Z]/go/src/github.com/rancher/distros-test-framework/shared/aux.go:318 github.com/rancher/distros-test-framework/shared.LogLevel Warning: systemctl start called with a glob pattern.
Hint: unit globs expand to loaded units, so start will usually have no effect.
   Passing --all will also load units which are pulled in by other units.
   See systemctl(1) for more details.
Job for rke2-server.service failed because the control process exited with error code.
See "systemctl status rke2-server.service" and "journalctl -xeu rke2-server.service" for details.
 
ERRO[2023-11-16T05:14:47Z]/go/src/github.com/rancher/distros-test-framework/shared/aux.go:298 github.com/rancher/distros-test-framework/shared.ReturnLogError Error starting rke2 service
Last call: github.com/rancher/distros-test-framework/pkg/testcase.certRotate in /go/src/github.com/rancher/distros-test-framework/pkg/testcase/certrotate.go:57
```

Reply:
cmd = fmt.Sprintf("timeout 2m sudo systemctl start --all %s*", product)

but only if you want, if you prefer stick with the function, thats nice also